app/views/customer_requests/_company_customer_requests.html.erb

```diff
@@ -13,6 +13,8 @@
        </thead>
        <tbody>
            <% @requests.each do |request| %>
+            <% unless request.expires_date < Date.today %>
```

Comment:
I really appreciate the use of unless, and ruby gives us that for a good reason. But here, `<% if request.expires_date > Date.today %>` makes more sense in ruby world because we aren't inferring that things get blacklisted, we're saying the BELONG here IF. We try to pinpoint the unique selection rather than dismissing common because that allows the reader to spend less time wondering why we're using the unless in the first place, its vexxing and just makes more sense as an if, LOL sorry for droning on